Shared/mods/deathmatch/logic/luadefs/CLuaCryptDefs.cpp

```diff
@@ -589,6 +596,150 @@ int CLuaCryptDefs::EncodeString(lua_State* luaVM)
                 }
                 return 1;
             }
+            case StringEncodeFunction::BASE64:
+            {
+                SString& variant = options["variant"];
+
+                if (!variant.empty())
+                {
+                    variant = variant.ToUpper();
+
+                    if (variant != "URL")
```

Comment:
Can be compacted down to 
```
const SString variant = options["variant"].ToUpper();
if (!variant.empty() && variant != "URL")
```
The same applies to the lines below.

Reply:
Good suggestion, done.